nextflow.config

```diff
@@ -50,10 +50,26 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
-    //StarAMR options
-    pointfinder_database             = null
-    plasmidfinder_database           = null
-    mlst_scheme                      = null
+    // StarAMR options
+
+    // Databases
+    pointfinder_database             = "Automatic Selection"
+    plasmidfinder_database           = "All"
+    mlst_scheme                      = "Automatic"
+
+    // Additional CLI arguments
+    genome_size_lower_bound              = 4000000
+    genome_size_upper_bound              = 6000000
+    minimum_N50_value                    = 10000
+    minimum_contig_length                = 300
+    unacceptable_number_contigs          = 1000
+    pid_threshold                        = 98
```

Comment:
This pid_threshold maybe doesn't need to change, but Aaron and I were discussing at one point to lower this to like 96 because of an issue with pbp5.

Reply:
Similar to previous comment. For now I was just porting over defaults users would be familiar with.

Reply:
We can discuss if we need to adjust the default in our meeting next week. Or, we could later implement something like adjusting the default value based on the species.